pages/app-layout/dashboard-content-type.page.tsx

```diff
@@ -23,12 +24,14 @@ export default function () {
         breadcrumbs={<Breadcrumbs />}
         navigation={<Navigation />}
         tools={<Tools>{toolsContent.long}</Tools>}
-        contentHeader={
```

Comment:
Refactored to stop using the deprecated `contentHeader` property. This makes it easier to find this header in the newly added motion test.